auto_nag/scripts/configs/tools.json

```diff
@@ -66,7 +66,11 @@
     "log": "/tmp/auto_nag_errors.txt",
     "receiver_list": {
       "rm": ["calixte@mozilla.com", "release-mgmt@mozilla.com"]
-    }
+    },
+    "needinfo_skiplist": [
+      "intermittent-bug-filer@mozilla.bugs",
+      "wptsync@mozilla.bugs"
```

Comment:
Both emails can be ignored using the `utils.is_no_assignee()` function:

https://github.com/mozilla/relman-auto-nag/blob/1857bdb559fc3757337ecb8c64fae24eb59d2dde/auto_nag/utils.py#L155-L156

It can be added as `ni_mail in self.get_auto_ni_skiplist() or not utils.is_no_assignee(ni_mail)` at:

https://github.com/mozilla/relman-auto-nag/blob/1857bdb559fc3757337ecb8c64fae24eb59d2dde/auto_nag/bzcleaner.py#L224

Reply:
Good idea, done in aa86af5.